site/zh-cn/tensorboard/.ipynb_checkpoints/tensorboard_profiling_keras-checkpoint.ipynb

```diff
@@ -0,0 +1,1051 @@
+{
```

Comment:
This file should not be checked in and I thought it was covered by this [.gitignore](https://github.com/tensorflow/docs-l10n/blob/master/.gitignore#L5) rule?

Reply:
look like my git meet some problems，i have deleted this file